packages/get-starknet/src/wallet.ts

```diff
@@ -55,6 +61,16 @@ export class MetaMaskSnapWallet implements StarknetWindowObject {
 
   lock: MutexInterface;
 
+  #pollingController: AbortController | undefined;
+
+  #accountChangeHandlers: AccountChangeEventHandler[] = [];
+
+  #networkChangeHandlers: NetworkChangeEventHandler[] = [];
+
+  static readonly minimumPollingDelay = 100;
+
+  static readonly pollingTimeoutMs = 5000;
```

Comment:
nit:
static var is similar with constants, lets use upper case at the first char (if the lint allow)
```suggestion
  static readonly PollingDelayMs = 100;

  static readonly PollingTimeoutMs = 5000;
```

Reply:
lint does not allow start with caps but I renamed the constants with your suggestion